pkg/genericapiserver/genericapiserver.go

```diff
@@ -128,6 +131,20 @@ type APIGroupVersionOverride struct {
 	ResourceOverrides map[string]bool
 }
 
+// Info about an API group.
+type APIGroupInfo struct {
+	GroupMeta latest.GroupMeta
```

Comment:
Yes. How about I move it to unversioned now and then we can move all types away from unversioned together?
Or if you have a better place in mind, I can move it there directly.